vdist/defaults.py

```diff
@@ -1,7 +1,8 @@
 import os
 import sys
 
-PYTHON_BASEDIR = '/opt/vdist-python'
+PYTHON_BASEDIR = '/opt'
+COMPILED_PYTHON_FOLDER_NAME = 'vdist-python'
```

Comment:
Nit picky, but could you please rename this to COMPILED_PYTHON_DIR_NAME instead? So that we consistently use "DIR" instead of "FOLDER".